services/agents/node.go

```diff
@@ -29,6 +29,33 @@ import (
 	"github.com/percona/pmm-managed/utils/collectors"
 )
 
+var v2_26_99 = version.MustParse("2.26.99")
```

Comment:
I suggest to put all logic related to some version in one place. so it's easier to manage it.

Reply:
you mean declarations only? because for all version checks it would be quite a lot of changes, almost every entity within that package has some backward-compatibility checks

Reply:
resolve it temporary and move to another ticket

Reply:
Ok, now I remember why I wrote it.
We have multiple checks of pmmAgentVersion here in code, so it'll easier to read if we have one place with all code related to some version there.

Reply:
@tshcherban this one is missed